backend/EkoFunkcje/EkoFunkcje/Features/Comments/DeleteCommentFunction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AzureFunctions.Extensions.Swashbuckle.Attribute;
+using EkoFunkcje.Models;
+using EkoFunkcje.Models.Requests;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.WindowsAzure.Storage.Table;
+using Microsoft.Extensions.Logging;
+
+namespace EkoFunkcje.Features.Comments
+{
+    public static class DeleteCommentFunction
+    {
+        [FunctionName("DeleteComment")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "DeleteComment")]
+            [RequestBodyType(typeof(DeletionRequest), "DeletionRequest")]DeletionRequest request,
+            [Table(Config.CommentsTableName, Connection = Config.StorageConnectionName)]CloudTable commentsTable,
+            ILogger log)
+        {
+            var results = await commentsTable.ExecuteQuerySegmentedAsync(
```

Comment:
to zalezy jak ta funkcje rozwiazemy i czy beda komentarze dodawane do interwencji czy osobno

Reply:
Poprawione na dodawanie do interwencji